### PR TITLE
Require flag to track ip and user agent

### PR DESCRIPTION
**Pull request by NathanielWroblewski**

### CODE REVIEW

File: lib/mapper.js [hunk 1]
```diff
@@ -24,13 +24,17 @@ exports.identify = function(identify){
   var userId = identify.userId();
   var anonymousId = identify.sessionId();
   var payload = {};
+  var ip = identify.ip();
+  var shouldTrackIp = identify.proxy('context.trackIp');
```
Comment (percyhanna) [inline lib/mapper.js]: `context.Kissmetrics.trackIp`

**Comment by NathanielWroblewski** [comment]
@percyhanna looks like we need to add user agent to the other calls, and I'm going to make the following changes as well:

```
var shouldTrackIp = event === 'visited site' || context.Kissmetrics.trackIp
```

**Comment by percyhanna** [comment]
Looks good to me. :ship:

**Comment by NathanielWroblewski** [comment]
Going to merge here then open against segment.

**Pull request merged**